src/crypto.rs

```diff
@@ -164,8 +164,12 @@ pub(crate) fn asym_verify(
     signature: &str,
 ) -> Result<bool> {
     let mut verifier = Verifier::new(MessageDigest::sha256(), keypair)?;
+    verifier.set_rsa_padding(Padding::PKCS1_PSS)?;
+    verifier.set_rsa_mgf1_md(MessageDigest::sha256())?;
+    verifier
+        .set_rsa_pss_saltlen(openssl::sign::RsaPssSaltlen::MAXIMUM_LENGTH)?;
     verifier.update(message.as_bytes())?;
-    Ok(verifier.verify(signature.as_bytes())?)
+    Ok(verifier.verify(&base64::decode(signature.as_bytes()).unwrap())?)
```

Comment:
I think calls that may end up with a panic are prohibited in this project (you could add `// #[allow_ci]` comment). In this case, however, as the function returns `Result`, I guess it could be simply written with `?`:
```suggestion
    Ok(verifier.verify(&base64::decode(signature.as_bytes())?)?)
```

Reply:
OK, I'll make the change as suggested.